src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -62,6 +62,21 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        /// <summary>
+        ///     Gets or sets a timeout in milliseconds for a request to sources
+        /// </summary>
+        public int RequestTimeout { get; set; } = defaultRequestTimeout;
```

Comment:
```suggestion
        public TimeSpan RequestTimeout { get; set; } = defaultRequestTimeout;
```
I think using `TimeSpan` is much easier to use as the string representation / parsing is usable without the user needing to calculate 'milliseconds'.

Reply:
I'm not sure about this. According to [TimeSpan](https://learn.microsoft.com/en-us/dotnet/api/system.timespan.parse?view=net-8.0), it accept strings like `00:00:10`. Requesting timeout is mostly below one minute I believe. Such timeout string is too much more expressive than needed. Maybe use 'seconds' instead of 'milliseconds' in both `Preferences` and xml config?

Reply:
I'm not sure if the normal C# / unity developer has the knowledge of the string format of `TimeSpan` but it supports even nanoseconds precision with strings like `12:14:45.3448`. Yes a default like `00:01:30` so one and a half minute could eventually be misunderstood as 1 and a half hour. The alternative would be to use seconds but than we need to name the field `RequestTimeoutSeconds` so it is clear to the user what's the 'unit'. What do you think?